tools/builder/MINGW.sh

```diff
@@ -263,10 +263,10 @@ zlib_dist=$(table_line DISTS zlib-target)
 
 table_line_remove  DISTS zlib-target
 
-table_insert_after DISTS cmake "zlib-target $zlib_dist"
+table_insert_after DISTS curl "zlib-target $zlib_dist"
```

Comment:
Are these changes related?

Reply:
Yes, because I reordered the base dists list.